draft-jholland-quic-multicast.md

```diff
@@ -331,8 +331,11 @@ Under sustained persistent loss that exceeds server-configured thresholds, the s
 Under sustained persistent loss that exceeds client-configured thresholds, the client SHOULD reduce its Max Rate and tell the server via MC_LIMITS frames, which also will result in the server instructing the client to leave channels until the clients aggregate rate is below its advertised Max Rate.
 Under a higher threshold of sustained persistent loss, the client also SHOULD leave channels, using an MC_STATE(Left) frame with the High Loss reason, as well as reducing the Max Rate in MC_LIMITS.
 
-The unicast congestion control is unaffected.
-However if the client notices congestion from unicast this MAY also drive reductions in the client's Max Rate, and a lack of unicast congestion under unicast load MAY also drive increases to the client's Max Rate (along with an updated MC_LIMITS frame).
+The unicast connection's congestion control is unaffected.
+However a few potential interactions with the unicast connection are worth highlighting:
+
+ - if the client notices high loss on the unicast connection while multicast channel packets are arriving, the client MAY leave channels with reason High Loss (whether or not the arriving packets are decodable, in case the client is missing MC_INTEGRITY frames due to the high unicast loss)
```

Comment:
I think the comment in the parentheses is making this a bit confusing to me, I think it would be fine to leave it out.

I also now wonder if the client unilaterally leaving channels might cause more issues than it solves as the server probably has a better understanding on what channels to leave in what order. Maybe the client should simply reduce its max rate which will lead to the server sending leave frames? Not sure, but I think its fine for now and we can think more about it in a later rev.

Reply:
While I agree the server has a better view, I think there's risk that it takes a long time with an impaired network if the unicast connection is suffering loss and can't get the MC_LIMITS sent quickly or the MC_LEAVE received quickly, and it's also no different than the client refusing any particular channel it chooses to refuse or the network refusing to forward a channel.  After leaving and reducing limits, the server can still tell the client to leave one channel and join another, and failing to optimize which channel was best to leave probably just means some extra recovery or something.

I'll take out the parens, thanks.